packages/attestation-service/migrations/20201006231054-attestation-v6.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
```

Comment:
can consolidate these migration files

Reply:
i'm guessing you may have already deployed this to a network, so I won't change this